### PR TITLE
Add a lot of commands to get wholesome, dank, anime

### LINKED ISSUE #2: Command to send wholesome meme, dank meme, animememe
Something like `send wholesome` or `send wmeme` or send `send wholesome-meme` 

`send dank` and `send mimi`

### PR DESCRIPTION
**Pull request by dinhanhx**
send wholesome
send hot wholesome
send dank
send hot dank
send anime
send hot anime
send kemonomimi
send hot kemonomimi

issue to link #2

### CODE REVIEW

**Comment by dinhanhx** [comment]
It's not much bunch ctrl c and ctrl v

**Comment by Huy-Ngo** [comment]
You're supposed to say resolve #2

**Comment by dinhanhx** [comment]
Oh me is dummy

File: index.js [hunk 1]
```diff
@@ -33,8 +33,55 @@ client.on('message', msg => {
...
+            port: config.apiPort,
+            path: 'goodanimemes/new'
+        }, sendMeme)
+    } else if (message === 'send hot anime') {
+        http.get({
+            hostname: config.apiHost,
+            port: config.apiPort,
+            path: 'goodanimemes/hot'
+        }, sendMeme)
+    } else if (message === 'send kemonomimi') {
```
Comment (Huy-Ngo) [inline index.js]: Can you rename these to something easier to remember?

File: index.js [hunk 2]
```diff
@@ -33,8 +33,55 @@ client.on('message', msg => {
...
+            port: config.apiPort,
+            path: 'wholesomememes/new'
+        }, sendMeme)
+    } else if (message === 'send hot wholesome') {
+        http.get({
+            hostname: config.apiHost,
+            port: config.apiPort,
+            path: 'wholesomememes/hot'
+        }, sendMeme)
+    } else if (message === 'send dank') {
```
Comment (Huy-Ngo) [inline index.js]: Edit this to `send dank meme` please.

File: index.js [hunk 3]
```diff
@@ -33,8 +33,55 @@ client.on('message', msg => {
             port: config.apiPort,
             path: 'memes/hot'
         }, sendMeme)
+    } else if (message === 'send wholesome') {
```
Comment (Huy-Ngo) [inline index.js]: Edit this to `send wholesome meme` please

File: index.js [hunk 4]
```diff
@@ -33,8 +33,55 @@ client.on('message', msg => {
...
+            port: config.apiPort,
+            path: 'dankmemes/new'
+        }, sendMeme)
+    } else if (message === 'send hot dank') {
+        http.get({
+            hostname: config.apiHost,
+            port: config.apiPort,
+            path: 'dankmemes/hot'
+        }, sendMeme)
+    } else if (message === 'send anime') {
```
Comment (Huy-Ngo) [inline index.js]: The bot is not sending actual anime isn't it? It'd be more intuitive for the command to be "send anime meme" or something.

**Review by Huy-Ngo — CHANGES_REQUESTED** [review]
Please also add these new commands to README

**Comment by dinhanhx** [comment]
@Huy-Ngo I have changed cmds by request

File: README.md [hunk 5]
```diff
@@ -26,3 +26,27 @@ if you find that the current API doesn't fulfill your planned feature.
...
+send hot dank meme
+
+send anime meme
+
+send hot anime meme
+
+send girl with mini
+
+send hot girl with mini
+```
```
Comment (Huy-Ngo) [inline README.md]: I mean adding at the Commands section.

**Review by Huy-Ngo — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (dinhanhx) [reply README.md]: I have refactored the cmd section as request

File: README.md [hunk 6]
```diff
@@ -20,7 +28,7 @@ Please read the [discord.js documentation](https://discord.js.org/#/docs/main/st
 
 ## API
 
-API used for meme bot has its source code at: [https://github.com/Huy-Ngo/reddit-meme-api](Reddit Meme API). Please contribute to that project
+API used for meme bot has its source code at: [Reddit api meme](https://github.com/Huy-Ngo/reddit-meme-api). Please contribute to that project
```
Comment (Huy-Ngo) [inline README.md:31]: Good catch lol

**Review by Huy-Ngo — APPROVED** [review]

**Pull request merged**